source/module_base/tool_check.h

```diff
@@ -1,28 +1,57 @@
-#ifndef TOOL_CHECK_H 
-#define TOOL_CHECK_H 
+#ifndef TOOL_CHECK_H
+#define TOOL_CHECK_H
 
-#include <vector>
-#include <valarray>
-#include <string>
+#include <cassert>
+#include <complex>
 #include <fstream>
+#include <iomanip>
 #include <iostream>
 #include <sstream>
-#include <iomanip>
-
-#include <complex>
-#include <cassert>
+#include <string>
+#include <valarray>
+#include <vector>
 
 namespace ModuleBase
 {
+/**
+ * @brief Check the next input from ifs is std::string.
+ * This is a global function.
+ *
+ * @param ifs The input file stream
+ * @param name_in The name for checking
+ * @param quit Whether call WARNING_QUIT to quit or not
+ */
+void CHECK_NAME(std::ifstream &ifs, const std::string &name_in, bool quit = true);
```

Comment:
For functions returning `void`, I suggest mark params out with [in] or [out]. See https://www.doxygen.nl/manual/commands.html#cmdparam

Reply:
OK, I will follow this in more complex functions.